test/components/views/beacon/BeaconViewDialog-test.tsx

```diff
@@ -103,23 +97,23 @@ describe("<BeaconViewDialog />", () => {
         const room = setupRoom([defaultEvent]);
         const beacon = room.currentState.beacons.get(getBeaconInfoIdentifier(defaultEvent))!;
         beacon.addLocations([location1]);
-        const component = getComponent();
-        expect(component.find("Map").props()).toEqual(
-            expect.objectContaining({
-                centerGeoUri: "geo:51,41",
-                interactive: true,
-            }),
-        );
-        expect(component.find("SmartMarker").length).toEqual(1);
+        getComponent();
+        // centered on default event
+        expect(mockMap.setCenter).toHaveBeenCalledWith({
+            lon: 41,
+            lat: 51,
+        });
+        // marker added
+        expect(mockMarker.addTo).toHaveBeenCalledWith(mockMap);
     });
 
     it("does not render any own beacon status when user is not live sharing", () => {
         // default event belongs to alice, we are bob
         const room = setupRoom([defaultEvent]);
         const beacon = room.currentState.beacons.get(getBeaconInfoIdentifier(defaultEvent))!;
         beacon.addLocations([location1]);
-        const component = getComponent();
-        expect(component.find("DialogOwnBeaconStatus").html()).toBeNull();
+        const { queryByText } = getComponent();
```

Comment:
Similar to my above point, if we had `screen` imported then we wouldn't need to destructure the return from the `getComponent()` calls as much as we do (at least not for queries, it would still need to be done for the cases that use `container`).

This might be straying into personal preference but I think avoiding destructuring for queries and using `screen` is the more "RTL" way of doing tests.

Reply:
That makes sense, I didn't know screen was recommended. Overall though this feels like a style thing, and not worth making the change now.